src/app/components/SearchResults/ResultsMetadata.jsx

```diff
@@ -17,7 +17,7 @@ const ResultsMetadata = ({ searchQuery, metadata: { total } }) => {
     lastElement = total;
   }
   if (total > 0) {
-    message = `Viewing ${firstElement} - ${lastElement} of ${total} items`;
+    message = `Viewing ${firstElement.toLocaleString()} - ${lastElement.toLocaleString()} of ${total.toLocaleString()} items`;
```

Comment:
I didn't know you could localize numbers in this way!

Reply:
Definitely a cool feature. Specific to the browser's locale? If so, do all browsers support this?

Reply:
Yes, it's specific to the browser locale, although we can get it fixed to 'en-US' if we want. 
Supported in all browsers

Reply:
No need to fix it to our way. It's always nice to see things in one's own particular idiom.